### DISCUSSION TITLE
Should the setup.sh script update to recognize and allow Xcode ^4.12?

### DISCUSSION THREAD

**Discussion by coldenate** [ianyh/Amethyst#1482] [Ideas]
Hai! ![](https://cdn.betterttv.net/emote/6176032f1f8ff7628e6ae13f/1x.png)

I’m super new to this repo and Swift development in general, so forgive me if this is a silly question. I was wondering if we should update the ﻿setup.sh script to recognize and allow Xcode ^4.12? 
![](https://cdn.betterttv.net/emote/62850a253c6f14b68848212a/1x.gif)
I’m generally curious about this and wanted to learn why the XCode versions matter so drastically as I have seen this in other repos.

Thanks in advance for your help!

**Comment by ianyh** [comment]
It should be updated, yes. Sometimes there are swift version incompatibilities, but those are honestly pretty rare these days.
